src/components/numeral-date/numeral-date.component.tsx

```diff
@@ -170,8 +298,13 @@ const NumeralDate = ({
   };
 
   const internalMessage = Object.keys(internalMessages).reduce(
-    (string, key) =>
-      internalMessages[key] ? `${string + internalMessages[key]}\n` : string,
+    (combinedMessage, datePart) =>
+      internalMessages[datePart as keyof NumeralDateObject]
```

Comment:
Same as the type assertion comment on `Object.fromEntries`

Reply:
date part will always be a key of the `NumeralDateObject`, this assertion is here to satisfy the compiler.